src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -23,8 +24,21 @@
 from vivarium_gates_nutrition_optimization_child.utilities import get_random_variable
 
 
-class ChildWasting:
+class ChildWasting(Component):
+    @property
+    def columns_required(self) -> Optional[List[str]]:
+        return ["alive", "age", self.dynamic_model.state_column]
+
+    @property
+    def initialization_requirements(self) -> Dict[str, List[str]]:
+        return {
+            "requires_columns": ["sex", self.static_model.propensity_column_name],
+            "requires_values": [self.static_model.exposure_pipeline_name],
+            "requires_streams": [],
+        }
+
     def __init__(self):
+        super().__init__()
```

Comment:
Aren't we missing something to set the Dynamic and Static wasting components as subcomponents?

Reply:
We have 

    def sub_components(self):
        return [
            self.dynamic_model,
            self.static_model,
        ]
        

(with the property decorator). Is there anything else we need?

Reply:
No, that's correct. That property just wasn't where I was just expecting it to be in the file.